src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -634,6 +641,13 @@ protected function initializeContainer()
         try {
             $container = null;
             $container = $this->buildContainer();
+            if ($asFiles) {
+                $container->getParameterBag()->add([
+                    'container.build_id' => 'static',
+                    'container.build_hash' => 'static',
+                    'container.as_files' => true,
```

Comment:
why is this needed? this should already be dumped as part of the regular container *when the param is defined*

Reply:
This is needed to avoid adding an extra param to the method `dumpContainer()`

Reply:
Fixed